locale/en_US/common.xml

```diff
@@ -453,6 +453,7 @@
 	<message key="notification.unsubscribeError">There was an error unsubscribing you.  Please ensure that you are using the correct URL as included in your notification email.</message>
 	<message key="notification.unsubscribeSuccess">You were successfully unsubscribed.</message>
 	<message key="notification.savedSubmissionMetadata">Submission metadata saved.</message>
+	<message key="search.author">By Author</message>
```

Comment:
Hey @NateWr , when you move keys from one file/repo to another, it would be really good to then move those keys in all existing locales. I fear that otherwise translators might end up with double work, deleting and re-translating at the other location.